pkg/api/avatar/avatar.go

```diff
@@ -119,7 +122,9 @@ func (this *CacheServer) Handler(ctx *macaron.Context) {
 	if avatar.notFound {
 		avatar = this.notFound
 	} else {
-		this.cache.Add(hash, avatar, gocache.DefaultExpiration)
+		if err := this.cache.Add(hash, avatar, gocache.DefaultExpiration); err != nil {
+			log.Warn("Error adding avatar to cache: %s", err)
```

Comment:
Need to be `Warnf` to work. But use argument `log.Warn("...", "err", err` instead.